docker-compose/op-scim.service

```diff
@@ -0,0 +1,17 @@
+#/etc/systemd/system/op-scim.service 
+[Unit]
+Description=1Password SCIM bridge
+Requires=docker.service
+After=docker.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+WorkingDirectory=/path/to/directory/containing/docker-compose.yml
+ExecStart=/usr/local/bin/docker-compose up -d
```

Comment:
can be used with `-f` to specify yml file

Reply:
thanks. since we name our example docker-compose.yml i think it's fine as is.